README.md

```diff
@@ -1390,28 +1390,27 @@ The Python SDK is built to work with Python 3.9 and newer. It is built using
 
 To build the SDK from source for use as a dependency, the following prerequisites are required:
 
-* [Python](https://www.python.org/) >= 3.9
-  * Make sure the latest version of `pip` is in use
+* [uv](https://docs.astral.sh/uv/)
 * [Rust](https://www.rust-lang.org/)
 * [Protobuf Compiler](https://protobuf.dev/)
-* [poetry](https://github.com/python-poetry/poetry) (e.g. `python -m pip install poetry`)
-* [poe](https://github.com/nat-n/poethepoet) (e.g. `python -m pip install poethepoet`)
 
-macOS note: If errors are encountered, it may be better to install Python and Rust as recommended from their websites
-instead of via `brew`.
+Use `uv` to install `poe`:
 
-With the prerequisites installed, first clone the SDK repository recursively:
+```bash
+uv tool install poethepoet
```

Comment:
We'll probably switch away from `poe` when `uv` [includes a task runner](https://github.com/astral-sh/uv/issues/5903).